ddtrace/internal/remoteconfig/worker.py

```diff
@@ -12,6 +12,9 @@
 
 DEFAULT_REMOTECONFIG_POLL_SECONDS = 5.0  # seconds
 
+# The RCM worker thread is started on start-up so we request soft fork hooks
+periodic.forksafe._soft = True
```

Comment:
Since this module is imported on `import ddtrace` we are effectively setting `forksafe._soft = True` for the whole package.

I think at this point we should just change the default value of `_soft` in `ddtrace.internal.forksafe` so that this is clear and avoid changing its value at the global level in any other module.

Reply:
I was thinking we could do this as part of the clean-up once we have the actual fix for gevent in place. WDYT?

Reply:
I'd prefer to do it now just to make the change obvious. It's pretty implicit as-is.